lib/private/Setup/PostgreSQL.php

```diff
@@ -94,6 +85,18 @@ public function setupDatabase($username) {
 			$this->createDatabase($connection);
 			// the connection to dbname=postgres is not needed anymore
 			$connection->close();
+
+			if ($canCreateRoles) {
```

Comment:
```suggestion
			if ($this->tryCreateDbUser && $canCreateRoles) {
```

It's possible to disable the user creation. 
If you do, the variables `$canCreateRoles` and `$connectionMainDatabase` are uninitialized.